stablehlo/reference/Ops.cpp

```diff
@@ -384,6 +384,16 @@ SmallVector<InterpreterValue> eval(Region &region,
       auto rhs = scope.findTensor(complexOp.getRhs());
       auto result = evalComplexOp(lhs, rhs, complexOp.getType());
       scope.add(complexOp.getResult(), result);
+    } else if (auto compositeOp = dyn_cast<CompositeOp>(op)) {
+      auto operands = scope.findTensors(compositeOp.getOperands());
+      SmallVector<InterpreterValue> values(operands.size());
+      std::transform(operands.begin(), operands.end(), values.begin(),
+                     [](const Tensor &t) { return InterpreterValue(t); });
```

Comment:
More common pattern in our codebase is:

```
llvm::to_vector(llvm::map_range(operands, [](const Tensor &t) { return InterpreterValue(t); }))
```

Reply:
Might not need to_vector given that eval wants an arrayref? Not sure how that map_range result can be used.

Reply:
Very nice, thanks! I couldn't find an easy way to use `ArrayRef` directly here, so used `to_vector`.